harvard-lts/etd-integration-tests
# Etd 161: alma service integration test

**Etd 161: alma service integration test.**
* * *

**JIRA Ticket**: https://at-harvard.atlassian.net/browse/ETD-161


# What does this Pull Request do?
adds integration test for the alma service

# How should this be tested?

1) copy env.example to .env and use dev settings
2) copy `alma_dropbox_id_rsa` & `known_hosts` from the vault and put in  `<repo>/.ssh`
3) start up container
4) go to https://localhost:10610/alma_service
5) all tests should pass.


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? n
- integration tests? n

# Interested parties
Tag (@ mention) interested parties: @awoods
